suite-native/transactions/src/components/TransactionsList/TransactionList.tsx

```diff
@@ -77,9 +77,12 @@ export const TransactionList = ({
     }
 
     const renderSectionHeader = useCallback(
-        ({ section: { dateKey } }: RenderSectionHeaderParams) => (
-            <TransactionListGroupTitle dateKey={dateKey} key={dateKey} />
-        ),
+        ({ section: { dateKey } }: RenderSectionHeaderParams) => {
+            const isPending = dateKey === 'pending';
```

Comment:
Why are you doing this check here? It will prefer just passing date key to component and do check inside of component

Reply:
I thought since it's prop it would rerender on new blockchain event if the pending transaction is confirmed. But it's not in the useCallback deps, so I guess it might as well be in the cmp, right?

Reply:
It is always rerendered with `sectionsData` changed which are created from `transactions` prop of this component. So your blockchain event about a confirmation of a transaction will be handled and can be handled inside of `<TransactionListGroupTitle>` comp.